manifest.xml

```diff
@@ -9,5 +9,4 @@
   <logo>http://</logo>
   <depend package="base/cmake" />
   <depend package="base/types" />
-  <depend package="drivers/hidapi" />
```

Comment:
All ok here:

`operating system: ubuntu,debian - 16.04,lts,xenial,xerus`

```
autoproj show libhidapi
the osdep 'libhidapi'
  apt-dpkg: libhidapi-dev
  is not directly selected by the manifest
  direct reverse dependencies: drivers/tempered
  recursive reverse dependencies: drivers/tempered
  selected by way of
    bir.flat_fish.br>drivers/tempered>libhidapi
```

Should we merge this PR?